main.go

```diff
@@ -1,22 +1,62 @@
 package main
 
 import (
+	"context"
 	"fmt"
 	"log"
 	"net/http"
 	"os"
 
+	"gitlab.com/NebulousLabs/errors"
+
 	"github.com/NebulousLabs/skynet-accounts/api"
+	"github.com/NebulousLabs/skynet-accounts/database"
+)
+
+var (
+	// envDBHost holds the name of the environment variable for DB host.
+	envDBHost = "SKYNET_DB_HOST"
+	// envDBPort holds the name of the environment variable for DB port.
+	envDBPort = "SKYNET_DB_PORT"
+	// envDBUser holds the name of the environment variable for DB username.
+	envDBUser = "SKYNET_DB_USER"
+	// envDBPass holds the name of the environment variable for DB password.
+	envDBPass = "SKYNET_DB_PASS" // #nosec G101: Potential hardcoded credentials
 )
 
+// dbConnFromEnv creates a new DB connection based on credentials found in the
+// environment variables.
+func dbConnFromEnv(ctx context.Context) (*database.DB, error) {
```

Comment:
(nit) I think having a `loadCredentials() database.DBCredentials` makes more sense and is easier to unit test separately, `dbConnFromEnv` looks a bit weird imo

Reply:
Done in #8 .